### ISSUE TITLE
Left click try icon skinned like osd

### ISSUE THREAD

**Issue by googanhiem** [irzyxa/Volume2#204]
I'm really enjoying the programme thanks @irzyxa. 
I really like the skin options for the osd. Is there anyway we could skin the "Standard Regulator" brought up with the "left button single click" on the tray icon to look more like the OSD, it currently seems a bit out of place.

**Comment by BenJewell** [comment]
I agree, I was just trying this out and noticed that. It should be customizable I think.

**Comment by Kolmarwow** [comment]
I second this, it'd be nice to have a left to right slider option in the Windows 10 v2 skin.
